bin/query-cloudwatch

```diff
@@ -235,6 +371,11 @@ class QueryCloudwatch
         config.format = :csv
       end
 
+      opts.on('--sqlite [FILENAME]', 'load output into the given Sqlite database (events.db by default). Output MUST include @message and @timestamp') do |filename|
+        config.format = :sqlite
+        config.sqlite_database_file = filename || 'events.db'
+      end
```

Comment:
so we have the `MAX_RESULTS_LIMIT_MATCHER` for some other logic that ensure there is a `| limit 10000` for `--complete` option, WDYT about moving the requirement to check for `@message` and `@timestamp` into the `cloudwatch_client.rb`?

Reply:
I went with just erroring out if the query results don't include these required fields. That ends up looking something like:

```shell
❯ aws-vault exec prod-analytics -- bin/query-cloudwatch --app idp --env prod --log events.log --from 3min  --sqlite --query "fields name | limit 10"         
Querying log slices 100% [1/1] |=====================================================================================================================================| Time: 00:00:03
Wrote 0 rows to the 'events' table in events.db
bin/query-cloudwatch:106:in `handle_row': Query must include @message in output when using --sqlite (RuntimeError)
        from bin/query-cloudwatch:191:in `block (2 levels) in run'
        from /Users/matthewghinz/Work/18f/identity-idp/lib/reporting/cloudwatch_client.rb:141:in `block in fetch'
```

Reply:
Makes sense! If I were coding it, I would choose to move the error earlier if possible, because it takes a minimum of 3 seconds for a CW result to come back, so I would want to provide feedback faster.